tools/benchmark.sh

```diff
@@ -463,7 +463,7 @@ function run_change {
   operation=$1
   echo "Do $num_keys random $operation"
   log_file_name="$output_dir/benchmark_${operation}.t${num_threads}.s${syncval}.log"
-  cmd="./db_bench --benchmarks=$operation \
+  cmd="./db_bench --benchmarks=$operation,waitforcompaction \
```

Comment:
Rather than running the two benchmarks ($operation, waitforcompaction) in a single run, would it be better to run them in two separate process instances  (like bulkload does)?  This way, the output results can still be compared without the additional compaction potentially skewing the results

Reply:
This case is different: here, we *do* want to "charge" the overwrite step for the additional compactions it triggers. In other words, the results are currently skewed (in that the compactions triggered by overwrite get charged to the next benchmark) and this would *un*skew them. (Note that we're adding a "waitforcompaction" step, which simply loops until there are no more pending compactions, unlike "compact" which actually issues a manual compaction request.)

Reply:
But what are you going to "charge" it to?  Will it change the mb/sec, pXX times, etc?  I see how it will change the total time of the operation, but it seems like it will more skew results than produce something that useful since the compaction is completed after all of the other operations are done.

Reply:
> But what are you going to "charge" it to?

My main focus is getting more accurate write amp values.

Reply:
P.S. It doesn't skew the percentiles etc. because those are taken from overwrite's statistics line. (waitforcompact does not print one.)

Reply:
So a couple of (fairly minor) points:
- Can you post something to this PR of a sample set of results/output before and after this change and which values you expected to be changed?
- What happens for "timed" operations?  If I said to do 90 minutes of overwrite, now it is 90 minutes+compaction time.  How much time is that "likely" to add?
- All of the statistics are from a grep in the same output file (with some potential reformat).  Are you suggesting some values will not be updated by the "wait for compaction" (hence the value will be the last one printed via overwrite?). 

I understand the need for the change, but want to be sure we understand the ramifications of it.  Currently, the performance tests use a "cooling off overwrite" (rate limited) after the main one is complete to get the LSM back into a reasonable shape.  From this discussion, I am not sure if that step is still required or not, nor what this change does to the existing performance metrics posted (how valid/different after this change?  Is there any baseline still to compare to?, etc).